examples/Project.toml

```diff
@@ -32,10 +31,5 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 ThreadsX = "ac1d9e8a-700a-412c-b207-f0111f4b6c0d"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
-[compat]
-OrdinaryDiffEq = "5.64.0"
```

Comment:
How is this suddenly not needed anymore? I thought that there were interface changes we needed to upgrade for the implicit solver. Are we not running any implicit examples? If so, that’s fine.

Reply:
Yeah, i was just trying to get the most recent versions. I'm not sure how best to deal with this (we may have to bite the bullet and update everything)